terraform/terraform.tfvars

```diff
@@ -0,0 +1,31 @@
+# Generics
+region = "us-east-1"
+ 
+tags = [{ Project = "lanthorn",State = "Deploy EC2 with nvidia drivers and docker installed"}]
+
+
+# Networking
+vpc = [{ name = "lanthorn_vpc", cidr_block = "10.0.0.0/16" , instance_tenancy = "default", enable_dns_hostnames = true, enable_dns_support = true}]
+
+subnet_prefix = [{ name = "langthron_module_subnet", cidr_block = "10.0.1.0/24", availability_zone = "us-east-1a", map_public_ip_on_launch = true }]
```

Comment:
langthron_module_subnet -> lanthorn...

I wonder about all of these ids. They only work for a specific account right? So we will have to mention that they must be changed